internal/pkg/aws/cloudformation/stackstatus.go

```diff
@@ -9,15 +9,58 @@ import (
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 )
 
+var (
+	successStackStatuses = []string{
+		cloudformation.StackStatusCreateComplete,
+		cloudformation.StackStatusDeleteComplete,
+		cloudformation.StackStatusUpdateComplete,
+		cloudformation.StackStatusUpdateCompleteCleanupInProgress,
+		cloudformation.StackStatusImportComplete,
+	}
+
+	failureStackStatuses = []string{
```

Comment:
What do you think we display everything complete -> green, in progress -> yellow, failed -> red?

Reply:
We do that except for "inprogress -> yellow". I chose "fainted" on purpose so that while stuff is in progress it doesn't catch the eye. It's only when it becomes complete or a failure that it becomes a vibrant color.